k8s/scripts/kubelet-config-helper.sh

```diff
@@ -539,6 +544,7 @@ function config_kubelet() {
 	if [[ "$kubelet_env_files" == "" ]]; then
 		kubelet_env_file="/etc/default/kubelet"
 		touch "$kubelet_env_file"
+		sed -i "/^[Service]/a EnvironmentFile=/etc/default/kubelet" "$systemd_file"
```

Comment:
I don't quite understand this `sed`; can you show an example of the before and after?

Reply:
@ctalledo Basically it adds the detected /etc/default/kubelet env file as EnvironmentFile to the existing kubelet systemd unit (not part of it by default in AL2) after the [Service] section header:

```
[Service]
Slice=runtime.slice
EnvironmentFile=/etc/default/kubelet
ExecStartPre=/sbin/iptables -P FORWARD ACCEPT -w 5
ExecStart=/usr/bin/kubelet \
    --config /etc/kubernetes/kubelet/kubelet-config.json \
    --kubeconfig /var/lib/kubelet/kubeconfig \
    --container-runtime-endpoint unix:///run/containerd/containerd.sock \
    --image-credential-provider-config /etc/eks/image-credential-provider/config.json \
    --image-credential-provider-bin-dir /etc/eks/image-credential-provider \
    $KUBELET_ARGS \
    $KUBELET_EXTRA_ARGS \
  $KUBELET_CRIO_ARGS
```

Reply:
Ah got it, thanks for the clarification!